### DISCUSSION TITLE
求助：RangePicker 回填怎么做的

### DISCUSSION THREAD

**Discussion by yysfwls** [vbenjs/vue-vben-admin#2827] [Q&A]
求助：RangePicker 回填怎么做的

**Comment by yysfwls** [comment]
RangePicker 回填怎么做的到底咋做啊救命。。。。拆两个写太难受了

**Reply by ppd0705** [reply]
是不知道怎么用RangePicker吗？可以看下文档  https://doc.vvbin.cn/components/form.html#fieldmaptotime

我是在useTable 里面使用，示例如下，将`date_range` 映射成 `start_date` 和 `end_date`
```js
 formConfig: {
          labelWidth: 80,
          schemas: searchFormSchema,
          fieldMapToTime: [['date_range', ['start_date', 'end_date'], 'YYYY-MM-DD']],
 }
```
